platform/ui/src/components/SidePanelWithService/SidePanelWithService.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import SidePanel from '../SidePanel';
+import { PanelService, ServicesManager, Types } from '@ohif/core';
+
+const SidePanelWithService = ({
+  servicesManager,
+  side,
+  activeTabIndex: activeTabIndexProp,
+  tabs: tabsProp,
+  ...props
+}) => {
+  // Tracks whether this SidePanel has been opened at least once since this SidePanel was inserted into the DOM.
+  // Thus going to the Study List page and back to the viewer resets this flag for a SidePanel.
+  const [hasBeenOpened, setHasBeenOpened] = useState(
+    activeTabIndexProp !== null
+  );
+
+  const panelService: PanelService = servicesManager.services.panelService;
+  const initialTabsState = tabsProp ?? panelService.getPanels(side);
+  const [tabs, setTabs] = useState(initialTabsState);
```

Comment:
you can get rid of initialTabsState and reducing one line

Reply:
Updated